src/RL/representation/default/defaultstaterepresentation.jl

```diff
@@ -81,6 +81,6 @@ end
 
 Returns the length of the feature vector, for the `DefaultFeaturization`.
 """
-feature_length(::Type{<:FeaturizedStateRepresentation{DefaultFeaturization, TS}}) where TS <: DefaultTrajectoryState = 3
+feature_length(::Type{<:FeaturizedStateRepresentation{DefaultFeaturization, TS}}) where TS = 3
```

Comment:
I finally removed the rapid fix that we proposed yesterday. There was initially no reason to specify the parametric type `TS`, the error we encountered before could have been the result of an uncanny artifact due to conflict between updated dependencies and our older Julia v1.6...